src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/GetInstanceBatchesByTimeStamp.sql

```diff
@@ -0,0 +1,52 @@
+/***************************************************************************************/
+-- STORED PROCEDURE
+--     GetInstanceBatchesByTimeStamp
+--
+-- DESCRIPTION
+--     Divides up the instances into a configurable number of batches filter by timestamp.
+--
+-- PARAMETERS
+--     @batchSize
+--         * The desired number of instances per batch. Actual number may be smaller.
+--     @batchCount
+--         * The desired number of batches. Actual number may be smaller.
+--     @status
+--         * The instance status.
+--     @startTimeStamp
+--         * The start filter timestamp.
+--     @endTimeStamp
```

Comment:
Should end time be inclusive?

Reply:
It is inclusive, will update the comment

Reply:
My original thought was maybe this should be exclusive, but the use case for it here makes it reasonable. A scenario where an exclusive end time helps is when you're querying batches by time (e.g. [1:00, 2:00), [2:00, 3:00), [3:00, 4:00), etc)